development/idp2-local/config/config.php

```diff
@@ -46,7 +46,6 @@
 $ADMIN_NAME = Env::get('ADMIN_NAME', 'SAML Admin');
 $ADMIN_PROTECT_INDEX_PAGE = Env::get('ADMIN_PROTECT_INDEX_PAGE', true);
 $SHOW_SAML_ERRORS = Env::get('SHOW_SAML_ERRORS', false);
-$TIMEZONE = Env::get('TIMEZONE', 'GMT');
```

Comment:
Wouldn't it have been easier to hardcode this, rather than change all 'timezone' references below? What if the timezone code changes from GMT to something else in the future? More code changes.

Reply:
I sure hope it doesn't change. But you're right about it being easier the other way.